homeassistant/components/assist_satellite/entity.py

```diff
@@ -54,6 +61,67 @@ def vad_sensitivity_entity_id(self) -> str | None:
         """Entity ID of the VAD sensitivity to use for the next conversation."""
         return self._attr_vad_sensitivity_entity_id
 
+    async def async_internal_announce(
```

Comment:
Should we check the supported features here and raise an exception?

Reply:
No. That is done by the service call handler. The service is registered in `__init__.py` with the correct feature flag.